webapp-django/crashstats/topcrashers/views.py

```diff
@@ -66,11 +66,19 @@ def get_date_boundaries(parameters):
     return (greater_than, lower_than)
 
 
+def datetime_to_build_id(date):
+    """Return a build_id-like string from a datetime. """
+    return date.strftime('%Y%m%d%H%M%S')
+
+
 def get_topcrashers_results(**kwargs):
-    '''Return the results of a search. '''
+    """Return the results of a search. """
     results = []
 
     params = kwargs
+    build_mode = params.pop('_range_type') == 'build'
```

Comment:
Yes, I try to make this page's URL compatible with Super Search. In Super Search all parameters that are not filters start with an underscore (like ``_results_number`` or ``_sort`` for example). 

I think it's a mistake to even pass a boolean and not just the same variable. I'll change that.